### PR TITLE
deps: switch from thermite to halogen

### PR DESCRIPTION
**Pull request by lthms**
# QA

Assuming you start from project root

``` sh
# build the haskell world
stack build
# start gjanajo (a mock, but required)
stack exec gjanajo &
# start bz
stack exec bz &
# create an account (required, if not it fails)
curl http://localhost:11000/accounts/new -XPOST
# build nahand
cd nahand/
stack exec r2purs -- --bz --to src/
# start the launcher
npm install
electron .
```

If everything is right, you know have nahand, with the email input focused. You can type something, then hit down arrow to focus the submit button (up to refocus the input). Enter to select your email (which is currently ignored).

Once again, in case of success, an uuid is displayed: it should be the same as the result of the `curl` command.

### CODE REVIEW

**Comment by Nemikolh** [comment]
@lethom Please let me know once this is ready for review! :)

File: nahand/main.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ let win
 
 function createWindow () {
   // Create the browser window.
-  win = new BrowserWindow({width: 800, height: 600})
+  win = new BrowserWindow({width: 1800, height: 1600})
```
Comment (Nemikolh) [inline nahand/main.js]: Is it possible to set that to the screen size? I wonder what contains the `screen` object. Might be stored there.

Comment (lthms) [reply nahand/main.js]: I am not sure the launcher should be full screen, actually.

**Comment by lthms** [comment]
Okay, so it is not yet completely finished, but I’ve made some progress. As I suspected, halogen worthes the effort, event if there is not ugly types we have to deal with (this is the price to pay for type-safe UI, I guess).

I still have to have a proper error display, but you can try already if you want to! (at least, for the "best case scenario)

I updated the first post to explain what should be test and seen. I think, if you agree of course, we can merge that and do the error handling in another PR.

**Comment by lthms** [comment]
(I updated the QA notes, as bower and npm deps should be updated)

File: nahand/main.js (hunk 1 above)
Comment (Nemikolh) [reply nahand/main.js]: True. I guess what I wanted to say was more that you could chose a size that fit in the screen. The value you have chosen are quite big (my screen wouldn't contain a window that big).

Comment (lthms) [reply nahand/main.js]: That makes sense. I revert this change (I shouldn’t push it to begin with) and I will add an issue about that.

File: nahand/bower.json [hunk 2]
```diff
@@ -18,7 +18,8 @@
     "purescript-console": "^1.0.0",
     "purescript-servant-support": "^4.0.0",
     "purescript-thermite": "^2.0.0",
```
Comment (Nemikolh) [inline nahand/bower.json]: I guess you forgot to remove this line?

**Comment by Nemikolh** [comment]
@lethom In the QA notes, you can remove the bower and pulp part, they are run as part of the npm install. (Unless the change I've made are not part of that branch)

**Comment by lthms** [comment]
Thanks (:

**Review by Nemikolh — APPROVED** [review]
Ok! I have tried it and it works! :+1: :sparkles:

File: nahand/bower.json (hunk 2 above)
Comment (lthms) [reply nahand/bower.json]: Oh right, I’ll remove it right away.

**Pull request merged**